go.mod

```diff
@@ -161,3 +156,21 @@ replace k8s.io/client-go => k8s.io/client-go v0.26.3
 
 // Required until the following jira is resolved - https://issues.redhat.com/browse/MGDAPI-4996
 replace github.com/grafana-operator/grafana-operator/v4 => github.com/grafana-operator/grafana-operator/v4 v4.2.0
+
+//replace github.com/3scale/3scale-operator/apis => ../3scale-operator/apis
+
+replace github.com/3scale/3scale-operator/apis => github.com/valerymo/3scale-operator/apis v0.0.0-20230809091710-a1fe409c3f23
+
+replace (
```

Comment:
Hey,
Is this going to stay like that? I think we should not be making that many replaces, in general using replaces in RHOAM can lead to a lot of issues later on.